src/pages/CreateProtocolPage.jsx

```diff
@@ -111,7 +115,22 @@ function CreateProtocolPage(props) {
                 validation: [],
                 sugestions: [],
                 subForm: null,
-                id: null,
+                id: 1,
+            },
+        ]);
+    };
+
+    const handleSingleInputAdd = () => {
+        setInputs([
+            ...inputs,
+            {
+                description: '',
+                question: '',
+                type: 0,
+                validation: [],
+                sugestions: [],
+                subForm: null,
+                id: 3,
```

Comment:
```suggestion
                id: null,
            },
        ]);
    };

    const handleSingleInputAdd = () => {
        setInputs([
            ...inputs,
            {
                description: '',
                question: '',
                type: 2,
                validation: [],
                sugestions: [],
                subForm: null,
                id: null,
```
- A API identifica os tipos de campo pelo atributo _type_. Aqui, _type_ 0 daria a entender caixa de texto. Seleção única tem _type_ 2. Ademais, seja id uma chave primária com auto-incremento na API, convém não definir ids fixos (na criação ele provavelmente ignoraria o atributo, mas geraria problemas na edição, para a qual essa página também é usada).

Reply:
Bem observado :D